frontend/src/metabase-lib/lib/metadata/metadata.d.ts

```diff
@@ -1,36 +0,0 @@
-// to help declaring nominal types
-interface Flavoring<FlavorT> {
```

Comment:
Why is this one deleted?

Reply:
As Metadata.js is now Metadata.ts, you cannot actually have a .d.ts file for a ts file

Reply:
And it's not needed